rllib/tests/test_algorithm_save_load_checkpoint_learner.py

```diff
@@ -0,0 +1,90 @@
+import tempfile
+import unittest
+
+import ray
+from ray.rllib.utils.test_utils import check, framework_iterator
+from ray.rllib.algorithms.ppo import PPOConfig
+from ray.rllib.policy.sample_batch import DEFAULT_POLICY_ID
+from ray.rllib.utils.metrics.learner_info import LEARNER_INFO, LEARNER_STATS_KEY
+
+
+algorithms_and_configs = {
+    "PPO": (PPOConfig().training(train_batch_size=2, sgd_minibatch_size=2))
+}
+
+
+@ray.remote
+def save_and_train(algo_cfg, env, tmpdir):
+    algo_cfg = (
+        algo_cfg.training(_enable_learner_api=True)
+        .rl_module(_enable_rl_module_api=True)
+        .rollouts(num_rollout_workers=0)
+        .reporting(min_time_s_per_iteration=0, min_sample_timesteps_per_iteration=1)
+        .debugging(seed=10)
+    )
+    algo1 = algo_cfg.environment(env).build()
+
+    print(tmpdir)
+    tmpdir = str(tmpdir)
+    algo1.save_checkpoint(tmpdir)
+    print("======= training algo 1 =======")
+    algo1.train()
```

Comment:
Do you mean to run train() twice here? it's not explicit to the reader of this code what you want to do here. Also can you add docstring to everyone of these ray.remote utility tasks?